test/unit/specs/store/user.spec.js

```diff
@@ -50,10 +51,20 @@ describe(`Module: User`, () => {
   })
 
   it(`should show an error if loading accounts fails`, async () => {
+    let { actions, state } = userModule({
```

Comment:
shouldn't we define `actions` and `state` globally on line 13 ?

Reply:
I feel there's a lot of inconsistencies in our module tests, some of them use `mutations` and `actions` and others use `commit` and `dispatch`. We should clarify that in our coding guide and amend it with a refactor later on

Reply:
I oriented along the new coding guide (I wrote hehehe). I didn't want to refactor everything yet, just fullfill the issue. We can add a follow up to refactor the module tests according to the coding guide lines when you approve them.
Link to the guide lines: https://www.notion.so/cosmosvoyager/Code-Style-b974391ca44c450191b7719eb0235970